jsonforms-editor/src/text-editor/components/JsonEditorDialog.tsx

```diff
@@ -74,15 +76,37 @@ export const JsonEditorDialog: React.FC<JsonEditorDialogProps> = ({
   onCancel,
 }) => {
   const classes = useStyles();
-  const [content, setContent] = useState(initialContent);
+
+  const modelUri = Uri.parse('a://b/schema.json');
 
   const configureEditor = useCallback(
     (editor: EditorApi) => {
       if (type === 'JSON Schema') {
-        configureJsonSchemaValidation(editor);
+        configureJsonSchemaValidation(editor, modelUri);
+      }
+    },
+    [type, modelUri]
+  );
+
+  const model = useMemo(() => {
```

Comment:
Can we pull this out into some common util? we have the same code in https://github.com/eclipsesource/jsonforms-editor/pull/111/files#diff-7c0a057e15d46af8b81f5a00e9f63f09da18fa295a8498d50a525ca2cde488d2

Reply:
I created a helper method that both classes use.